docs/pages/components/breadcrumb.md

```diff
@@ -42,20 +42,22 @@ there should be popover with missing options.
                 </a>
             </div>
             <div class="fd-popover__body fd-popover__body--no-arrow" aria-hidden="true" id="breadcrumb1">
-                <ul class="fd-list fd-list--compact">
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
-                    </li>
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>                    
-                    </li>
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
-                    </li>
-                    <li class="fd-list__item">
-                      <a class="fd-list__title" href="#">Link Text</a>
-                    </li>
-                </ul>
+              <div class="fd-popover__body__no-overflow">
+                <ul class="fd-list fd-list--no-borderBot fd-list--compact">
```

Comment:
IMO no bottom border should be default behavior so no extra class is needed

Reply:
I will create another PR for this and put this one as blocked